src/pkg/input/string2string.go

```diff
@@ -0,0 +1,32 @@
+package input
+
+import (
+	"fmt"
+	"strings"
+)
+
+func StringToStringMap(pairs []string) (map[string]string, error) {
+	keyValMap := make(map[string]string)
+
+	for _, pair := range pairs {
+		keyValArr := strings.Split(pair, "=")
```

Comment:
Don't do string parsing yourself, let's find a different way to remove all labels.

Reply:
I will look for another library for the parsing